src/video-player.js

```diff
@@ -168,9 +174,12 @@ overrideDefaultVideojsComponents();
 let _allowUsageReport = true;
 
 class VideoPlayer extends Utils.mixin(Eventable) {
+
   constructor(elem, options, ready) {
     super();
 
+    this.isZoomed = false;
```

Comment:
Isn't there an object that contains all default configuration? better use such an object.

Reply:
It is not a configuration, it is a private variable. i will change it to '_ isZoomed' and if it will need to be public in the future we can add a getter function (to prevent changing it out of the class)